hs-ucan/library/Web/UCAN/Resolver/Class.hs

```diff
@@ -4,8 +4,7 @@ import           Network.IPFS.CID.Types
 
 import           RIO
 
-import qualified Web.UCAN.RawContent     as UCAN
 import           Web.UCAN.Resolver.Error as Resolver
 
 class Monad m => Resolver m where
-  resolve :: CID -> m (Either Resolver.Error UCAN.RawContent)
+  resolve :: CID -> m (Either Resolver.Error ByteString)
```

Comment:
There's a bunch of places that need the signature. We haven't resolved from CID before, so had never actually caught this. I'm 90% sure this is on me. Apologies 🙏

Reply:
I considered (and actually started on) making...

```haskell
newtype UnparsedUCAN = UnparsedUCAN Text
```

...just to tack the intention around in the code. Its' unparsed though, so `ByteString` kind of implies that we don't know what's in there already 🤷‍♀️ 

This could probably be replaced with good ol' IPFS.byCID (or whatever it's called), but the server and CLI call it differently so I kept it.

Reply:
I like this version.

The version before should be blamed on me! The type used to be `CID -> m (Either Resolver.Error (UCAN ..., RawContent))`. However that means you'd have to parse the UCAN at that point already, and that turned out difficult for reasons I don't quite remember anymore.

I misunderstood `RawContent` back then and assumed it was just a newtype wrapper for an "unparsed UCAN" like what the `ByteString` is now. But yeah, this is correct and I like it better :)

In fact, that's what I've changed it to in the `ucan-upgrade` branch:
https://github.com/fission-suite/fission/blob/c630610f53055a1d347b3672ffbbde3aaee6896f/hs-ucan/library/Web/UCAN/Resolver/Class.hs#L9-L10

Again, sorry for messing this up & not backporting it!